src/index.html

```diff
@@ -1,13 +1,42 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" data-qa="card" class="card">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="card__content">
+    <div class="card__wrapper">
+      <img
+        src="images/imac.jpeg"
+        alt="iMac"
+        class="card__img"
+      >
+      <div class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</div>
```

Comment:
```suggestion
      <div class="card__title">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </div>
```
И так во всем файле для читабельности (кроме очень коротких блоков)